site/index.pug

```diff
@@ -16,12 +16,40 @@ html(lang='en-US' dir="ltr").spectrum.spectrum--light.spectrum--medium
         include includes/sidebar.pug
 
         .spectrum-Site-mainContainer
-          .spectrum-Site-page
+          .spectrum-Site-page.spectrum-Typography
             .spectrum-Site-hero
               .spectrum-Site-heroHeading
                 h1.spectrum-Heading.spectrum-Heading--XXXL.spectrum-Heading--serif Meet Spectrum CSS
-              p.spectrum-Body.spectrum-Body--XL The CSS implementation of Adobe’s design system.
-              img.spectrum-Site-heroImage(src='img/spectrum_illustration_2x.png', alt='Spectrum Illustration')
+              p.spectrum-Body.spectrum-Body--XL Spectrum CSS is an open-source implementation of Spectrum, Adobe’s design system. It includes components and resources to make applications more cohesive.
+              img(alt="Spectrum CSS Hero image" src="img/spectrum-css_illustration_desktop@2x.png?w=976&amp;h=446" srcset="img/spectrum-css_illustration_desktop@2x.png 2x").spectrum-CenteredImage
+
+              div.spectrum-HomeCard
+                img(src="img/illustration_documentation.svg" alt="Documentation illustation")
+                div
+                  h4.spectrum-Heading.spectrum-Heading--S Robust documentation
+                  p.spectrum-Body.spectrum-Body--M Spectrum CSS is designed to be used in partnership with Spectrum’s detailed usage guidelines.
+                  p.spectrum-Body.spectrum-Body--M
+                    a.spectrum-Link(href="https://spectrum.adobe.com/" target="_blank") View Spectrum guidelines
+
+              div.spectrum-HomeCard
+                img(src="img/illustration_responsive.svg" alt="Responsove illustation")
+                div
+                  h4.spectrum-Heading.spectrum-Heading--S Flexible
+                  p.spectrum-Body.spectrum-Body--M Our CSS is customizable, powerful, and designed to work with any javascript framework.
+                  p.spectrum-Body.spectrum-Body--M
+                    a.spectrum-Link(href="get-started.html") Get Started
+
+              div.spectrum-HomeCard
+                img(src="img/illustration_tested.svg" alt="Tested illustation")
+                div
+                  h4.spectrum-Heading.spectrum-Heading--S Rigorously tested
+                  p.spectrum-Body.spectrum-Body--M These individually-versioned components have been vetted to be accessible and inclusive of global audiences.
+
+              div.spectrum-HomeCard
+                img(src="img/illustration_flexible.svg" alt="Flexible illustation")
+                div
+                h4.spectrum-Heading.spectrum-Heading--S Multi-platform support
+                  p.spectrum-Body.spectrum-Body--M We support evergreen browsers (minus one version) and IE 11 for scalability and flexibility.
```

Comment:
These little illustrations don't have the responsive layout that the existing site does, that will need to be added.